src/Sylius/Bundle/CoreBundle/Collector/SyliusCollector.php

```diff
@@ -120,7 +122,7 @@ public function collect(Request $request, Response $response, \Exception $except
 
             $this->data['base_currency_code'] = $channel->getBaseCurrency()->getCode();
             $this->data['currency_code'] = $this->shopperContext->getCurrencyCode();
-        } catch (CurrencyNotFoundException $exception) {}
+        } catch (ChannelNotFoundException $exception) {}
```

Comment:
I think it would be better to catch both exceptions, `$this->shopperContext->getCurrencyCode()` can also throw `CurrencyNotFoundException` even if there is a channel.